### PR TITLE
[TypeORM] Create TypeORM completion spec

### PR DESCRIPTION
**Pull request by EliasGcf**
Adds autocomplete spec for [TypeORM](https://typeorm.io/#/) CLI.

- Adds `typeorm` to yarn `cli` array.

### CODE REVIEW

File: dev/yarn.ts [hunk 1]
```diff
@@ -186,6 +186,7 @@ const completionSpec: Fig.Spec = {
       "eslint",
       "prettier",
       "tsc",
+      "typeorm",
```
Comment (fedeci) [inline dev/yarn.ts:189]: Can you add it also to the `npx` spec?

File: dev/typeorm.ts [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+      description: "Synchronizes your entities with database schema.",
+      options: [
+        cliOptions.help,
+        {
+          ...cliOptions.connection,
+          description:
+            "Name of the connection on which schema synchronization needs to to run.",
+        },
+        cliOptions.config,
+        cliOptions.version,
```
Comment (fedeci) [inline dev/typeorm.ts:31]: This does not need to be added to each subcommand.

File: dev/typeorm.ts [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+          name: ["-d", "--dir"],
+          description: "Directory where subscriber should be created.",
+          args: { name: "directory", template: "folders" },
+        },
+        cliOptions.config,
+        cliOptions.version,
+      ],
+    },
+    {
+      name: "migration:create",
```
Comment (fedeci) [inline dev/typeorm.ts:116]: `migrations:create` is an alias for this subcommand, same for the other migration commands.

**Review by fedeci — CHANGES_REQUESTED** [review]

File: dev/yarn.ts (hunk 1 above)
Comment (QuiiBz) [reply dev/yarn.ts:189]: Agree with @fedeci

File: dev/typeorm.ts [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+        {
+          name: ["-o", "--outputJs"],
+          description:
+            "Generate a migration file on Javascript instead of Typescript.",
+        },
+        cliOptions.version,
+      ],
+    },
+    {
+      name: "migration:generate",
```
Comment (QuiiBz) [inline dev/typeorm.ts:142]:
```suggestion
      name: ["migration:generate", "migrations:generate"],
```

File: dev/typeorm.ts (hunk 3 above)
Comment (QuiiBz) [reply dev/typeorm.ts:116]:
```suggestion
      name: ["migration:create", "migrations:create"],
```

File: dev/typeorm.ts [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+          name: ["-t", "--transaction"],
+          description:
+            "Indicates if transaction should be used or not for migration run.",
+        },
+        cliOptions.config,
+        cliOptions.version,
+      ],
+    },
+    {
+      name: "migration:show",
```
Comment (QuiiBz) [inline dev/typeorm.ts:198]:
```suggestion
      name: ["migration:show", "migrations:show"],
```

File: dev/typeorm.ts [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+        {
+          name: ["--ch", "--check"],
+          description:
+            "Verifies that the current database is up to date and that no migrations are needed.",
+        },
+        cliOptions.version,
+      ],
+    },
+    {
+      name: "migration:run",
```
Comment (QuiiBz) [inline dev/typeorm.ts:183]:
```suggestion
      name: ["migration:run", "migrations:run"],
```

**Review by QuiiBz — CHANGES_REQUESTED** [review]
That looks really great, thanks! Left some comments to resolve before merging it

File: dev/typeorm.ts (hunk 5 above)
Comment (EliasGcf) [reply dev/typeorm.ts:198]: It looks like `migrations:show` does not exist.

**Comment by EliasGcf** [comment]
Done! @QuiiBz

**Review by QuiiBz — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**